.github/workflows/ci.yml

```diff
@@ -41,6 +41,7 @@ jobs:
             collab-service,
             code-execution-service,
             communication-service,
+            qn-history-service,
```

Comment:
i think u need to set up the test environment and files first. bc i got errors running `npm test`